===== Patch instructions generated by SAVER =====
[+] { Insert: if (true) free(*(do_mkfs_exec_wrapper:mkfs)) at 20 (line 757, column 6),
        Insert: if (true) free(*(do_mkfs_exec_wrapper:mkfs)) at 23 (line 757, column 17) }
===== Actual patch in diff format =====
--- a/table1/benchmarks/lxc/src/lxc/bdev/bdev.c
+++ b/table1/benchmarks/lxc/src/lxc/bdev/bdev.c
@@ -755,6 +755,7 @@ int do_mkfs_exec_wrapper(void *args)
 
 	ret = snprintf(mkfs, len, "mkfs.%s", data[0]);
 	if (ret < 0 || (size_t)ret >= len)
+    free(mkfs);
 		return -1;
 
 	TRACE("executing \"%s %s\"", mkfs, data[1]);
